src/site/stages/build/process-cms-exports/transformers/block_content-promo.js

```diff
@@ -1,8 +1,12 @@
+const { getImageCrop } = require('./helpers');
+
 const transform = entity => ({
   entity: {
     entityType: 'block_content',
     entityBundle: 'promo',
-    fieldImage: { entity: entity.fieldImage[0] },
+    fieldImage: {
+      entity: getImageCrop(entity.fieldImage[0], '_32MEDIUMTHUMBNAIL'),
```

Comment:
Where is `_32MEDIUMTHUMBNAIL` coming from? I see it's one of the `mediaImageStyles` in `helpers`, but I'm not sure how we came up with this value in the first place. It also seems like it should be a constant.

Reply:
It's coming from the [GraphQL query](https://github.com/department-of-veterans-affairs/vets-website/blob/9757b4454077729edf67e17880c3620a7a9d60b4/src/site/stages/build/drupal/graphql/navigation-fragments/header.nav.graphql.js#L29) from the header